recipes/cmake-js/recipe.yaml

```diff
@@ -0,0 +1,39 @@
+context:
+  version: 7.3.0
+
+package:
+  name: cmake-js
+  version: ${{ version }}
+
+source:
+  url: https://registry.npmjs.org/cmake-js/-/cmake-js-${{ version }}.tgz
+  sha256: 92100f11132243e04099907991bdbbc98a62a685ad5f421164c15f88c43f7064
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  build:
+    - nodejs
+    - pnpm
+    - pnpm-licenses
+  run:
+    - nodejs
+
+tests:
+  - script:
+      - cmake-js --help
+
+about:
+  license: MIT
+  license_file:
+    - LICENSE
+    - third-party-licenses.txt
+  summary: CMake.js is a Node.js native addon build tool which works (almost) exactly like node-gyp, but instead of gyp, it is based on CMake build system.
```

Comment:
Shouldn't this then have a `run` dependency on `cmake`?

Reply:
Yes, that makes sense here.  Please see the last commit for that change.